docs/secret-rotation.md

```diff
@@ -0,0 +1,254 @@
+# Secret Rotations &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; **FluxCD**
+
+This brief guide should outline all the steps needed to be able to swap out a set of secrets or credentials with FluxCD GitOps.
+
+## [**Pre-requisites**](#pre-requisites-link)
+#### [Useful Resources](#useful-resources-link)
+#### [Applications & Software](#applications--software-link)
```

Comment:
Should these be inside the `Outline` section?

Reply:
I wouldn't say so, they aren't part of outlining the steps, or what is to come, and are things that are required before even starting.